view/frontend/web/js/recobuy.js

```diff
@@ -33,13 +33,16 @@
  *
  */
 
-define(['catalogAddToCart', 'nostojs', 'jquery'], function (addToCart, nostojs, $) {
+define([
+    'nostojs',
+    'jquery',
+    'catalogAddToCart'
```

Comment:
So we still need this?

Reply:
Yes, we need to require `catalogAddToCart` which will define a jQuery widget `catalogAddToCart `